src/main/java/org/jabref/logic/importer/fetcher/GoogleScholar.java

```diff
@@ -131,33 +130,36 @@ public String getName() {
     public List<BibEntry> performSearch(String query) throws FetcherException {
         try {
             obtainAndModifyCookie();
-            List<BibEntry> foundEntries = new ArrayList<>(10);
+            List<BibEntry> foundEntries = new ArrayList<>(20);
 
             URIBuilder uriBuilder = new URIBuilder(BASIC_SEARCH_URL);
             uriBuilder.addParameter("hl", "en");
             uriBuilder.addParameter("btnG", "Search");
             uriBuilder.addParameter("q", query);
 
-            addHitsFromQuery(foundEntries, uriBuilder.toString());
-
-            if (foundEntries.size() == 10) {
-                uriBuilder.addParameter("start", "10");
+            try {
                 addHitsFromQuery(foundEntries, uriBuilder.toString());
+
+                if (foundEntries.size() == 10) {
```

Comment:
Given the problems with the google fetcher, is it really wise to query it a second time for the next 10 results?

Reply:
Removed additional call.